app/src/main/java/com/example/home/models/Home.java

```diff
@@ -1,71 +1,56 @@
 package com.example.home.models;
 
+import org.json.JSONArray;
+import org.json.JSONException;
 import org.json.JSONObject;
 import com.parse.ParseClassName;
 import com.parse.ParseObject;
 
 @ParseClassName("Home")
 public class Home extends ParseObject {
-    private String mAddress;
-    private Double mHomePrice;
-    private Integer mHomeNoOfBedrooms;
-    private Integer mHomeNoOfBathrooms;
-    private Integer mHomeNoOfFloors;
-    private Integer mYearBuilt;
-    public boolean mSaved;
+    public String mAddress;
+    public String mPropertyType;
+    public String mHomeNoOfBedrooms;
+    public String mHomeNoOfBathrooms;
+    public String mYearBuilt;
+    public String mDistance;
 
     public Home(){}
 
-    public static Home fromJson(JSONObject jsonObject){
-        Home home = new Home();
-        return home;
+    public Home (JSONObject jsonObject) throws JSONException{
+        mAddress = jsonObject.getJSONObject("address").getString("oneLine");
```

Comment:
Let's have all the Json Param Names as constants in the class. Also, I'd prefer having this in a separate class called "HomeJsonParser"